test_snakes_cafe.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+import snakes_cafe
+
+def test_print_menu():
+    assert snakes_cafe.print_menu() == '''Menu:
+Appetizers
+
+Wings               2.00
+Calamari            2.00
+Spring Rolls        2.00
+Nachos              2.00
+Spinach Dip         2.00
+Sampler             2.00
+
+
+Entrees
+
+Salmon             10.00
+Steak              10.00
+Tacos              10.00
+Salad              10.00
+Pizza              10.00
+Vegetarian Option  10.00
+
+
+Sides
+
+French Fries        4.00
+Hush Puppies        4.00
+Green Beans         4.00
+Mashed Potatoes     4.00
+Corn                4.00
+Rolls               4.00
+
+
+Desserts
+
+Ice Cream           5.00
+Cake                5.00
+Pie                 5.00
+Cookies             5.00
+Cheese              5.00
+Boozy Milkshake     5.00
+
+
+Drinks
+
+Coffee              3.00
+Tea                 3.00
+Beer                5.50
+Soda                3.00
+Juice               3.00
+Evian               1.00\n\n'''
+
+def test_print_order():
+    assert snakes_cafe.print_order
```

Comment:
Just so you know, this test is currently only checking if the function `print_order` exists, which is not a good test.

Reply:
lol. i meant the first two tests that we wrote were working, not the first 2 tests in the file :)